docs/upgrade/troubleshooting.md

```diff
@@ -120,7 +120,7 @@ The Harvester controller deletes the upgrade repository VM and all files that ar
 3. Delete the Upgrade CR
 
     ```
-    $ kubectl delete hvst-upgrade-9gmg2 -n harvester-system
+    $ kubectl delete upgrade/hvst-upgrade-9gmg2 -n harvester-system
```

Comment:
Nice spot. For completeness, perhaps
```suggestion
    $ kubectl delete upgrade.harvesterhci.io/hvst-upgrade-9gmg2 -n harvester-system
```

Reply:
That's a great suggestion @bk201 - thanks for looking this over :smile: !
I've gone ahead and committed that suggestion on an amended commit.
I've also added you as co-author to the last commit as well :smile: !
Thanks for the help!
I figure this will definitely help others that read through the docs if for some reason an upgrade gets stuck.